src/stacky/stacky.py

```diff
@@ -1688,6 +1688,157 @@ def cmd_land(stack: StackBranchSet, args):
     cout("\n✓ Success! Run `stacky update` to update local state.\n", fg="green")
 
 
+def cmd_inbox(stack: StackBranchSet, args):
+    """List all active GitHub pull requests for the current user"""
+    fields = [
+        "number",
+        "title", 
+        "headRefName",
+        "baseRefName",
+        "state",
+        "url",
+        "createdAt",
+        "updatedAt",
+        "author",
+        "reviewDecision",
+        "reviewRequests"
+    ]
+    
+    # Get all open PRs authored by the current user
+    my_prs_data = json.loads(
+        run_always_return(
+            CmdArgs(
+                [
+                    "gh",
+                    "pr",
+                    "list",
+                    "--json",
+                    ",".join(fields),
+                    "--state",
+                    "open",
+                    "--author",
+                    "@me"
+                ]
+            )
+        )
+    )
+    
+    # Get all open PRs where current user is requested as reviewer
+    review_prs_data = json.loads(
+        run_always_return(
+            CmdArgs(
+                [
+                    "gh",
+                    "pr",
+                    "list",
+                    "--json",
+                    ",".join(fields),
+                    "--state",
+                    "open",
+                    "--search",
+                    "review-requested:@me"
```

Comment:
probably want assigned to me?

Reply:
Interesting yeah my current workflow only goes off of requests not assignments. I should probably include both here.

Reply:
Actually I don't want to add this. Assignee doesn't really desginate reviewer. Reviewer is an explicit field and that has state associated with it of request changes, waiting for review, and approved. With assignee it's hard to distinguish about particular reviewer state